test-alerts.cpp

```diff
@@ -3,6 +3,36 @@
 #include "test/catch.hpp"
 #include "typewise-alert.h"
 
-TEST_CASE("infers the breach according to limits") {
+TEST_CASE("infers the breach according to limits LOW") {
   REQUIRE(inferBreach(12, 20, 30) == TOO_LOW);
 }
+
+TEST_CASE("infers the breach according to limits HIGH") {
+  REQUIRE(inferBreach(32, 20, 30) == TOO_HIGH);
+}
+
+TEST_CASE("Test the Send operation - Controller") {
+  BatteryCharacter batterychar = {PASSIVE_COOLING, "ABC"};
+  REQUIRE( checkAndAlert(TO_CONTROLLER, batterychar, 25)== SEND_PASS);
+}
+
+TEST_CASE("Test the Send operation - Email") {
+  BatteryCharacter batterychar = {HI_ACTIVE_COOLING, "ABC"};
+  REQUIRE( checkAndAlert(TO_EMAIL, batterychar, 40)== SEND_PASS);
+}
+
+
+TEST_CASE("Test the Send operation - Email / High") {
+  BatteryCharacter batterychar = {HI_ACTIVE_COOLING, "ABC"};
+  REQUIRE( checkAndAlert(TO_EMAIL, batterychar, 50)== SEND_PASS);
```

Comment:
This REQUIRE statement conveys that checkAndAlert converts the given parameters to a SEND_PASS return value.
If you had to re-state its work in terms of data-conversions, what would you state?